src/App.css

```diff
@@ -140,6 +140,10 @@
 
 .section-card-info-text {
   width: 130px;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
```

Comment:
@DmitryApan pretty weird approach, why can't you use `max-height` and `overflow`?